runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -117,6 +122,18 @@
      */
     private final Logger correctnessLogger = LoggerFactory.getLogger("correctness");
 
+    /**
+     * Metrics registry and timer names used within VersionLockedObject methods
+     */
+    private static final MetricRegistry metrics = CorfuRuntime.getDefaultMetrics();
```

Comment:
Maybe it's a good idea to create  private static class `VersionLockedObjectMetrics` which will contain these fileds, and all relevant code for metrics?

Reply:
@xnull Done.